photo_points/Controllers/AdminController.cs

```diff
@@ -52,11 +59,28 @@ public IActionResult PhotoStream()
         [HttpGet]
         public IActionResult Pending()
         {
-            return View("Pending");
-            //all the new submissions can be displayed
-            // //  if Capture approval is false. 
-            // // display the Capture image and Data. 
+            //build a view model //
+            PendingViewModel pvm = new PendingViewModel();
+
+           // _adminReviewServices.GetUnapprovedCaptures(); ////the repository is calling the getunapproved captures
 
+            IEnumerable<Capture> pendingImages = _adminReviewServices.GetUnapprovedCaptures();
```

Comment:
This looks good, though your variable name would be clearer if you called it "pendingCaptures" instead of "pendingImages"